lib/rspec/support/object_formatter.rb

```diff
@@ -88,6 +88,21 @@ def pretty_print(pp)
           pp.text inspection
         end
       end
+
+      # @private
+      DelegatingInspector = Struct.new(:object) do
+        def inspect
+          if defined?(::Delegator) && ::Delegator === object
+            "#<#{object.class}(#{DelegatingInspector.new(object.__getobj__).inspect})>"
```

Comment:
Instead of just calling `inspect` on the wrapped object, it would be nice if the object was sent back through `prepare_for_inspection` above so that if you are wrapping one of the types of objects we have custom formatting for, that custom formatting will be displayed.  (And a spec for this case -- such as a delegator wrapping an array of time objects -- would be great, too).